test/argent-account-test.ts

```diff
@@ -72,6 +72,21 @@ describe("Argent account", function () {
         expect(balance).to.equal(initialBalance + 30n);
     });
 
+    it("should estimate deploy account fee and deploy with implicit maxFee", async function () {
+        const account = await starknet.ArgentAccount.createAccount({
+            salt: "0x43",
+            privateKey: argentAccountPrivateKey
+        });
+
+        const estimatedFee = await account.estimateDeployAccountFee();
```

Comment:
We aren't really relying on the fee estimation. I would suggest that in this test-case we use the estimated fee both for minting and as the value for maxFee of deployment.

In the other test-case we can rely on implicit fee estimation (and mint 1e18 before that).

Reply:
https://github.com/Shard-Labs/starknet-hardhat-example/pull/96/commits/b5985e96e87a88eab4d32e8373e5f0931a2c5c60